recipes/libbigwig/all/test_package/CMakeLists.txt

```diff
@@ -1,8 +1,10 @@
 cmake_minimum_required(VERSION 3.8)
-project(test_package C)
+project(test_package LANGUAGES C)
+
+set(CMAKE_C_STANDARD 11)
 
 find_package(BigWig REQUIRED CONFIG)
 
 add_executable(${PROJECT_NAME} test_package.c)
 target_link_libraries(${PROJECT_NAME} PRIVATE BigWig::BigWig)
-target_compile_features(${PROJECT_NAME} PRIVATE c_std_11)
+target_compile_features(${PROJECT_NAME} PRIVATE c_std_${CMAKE_C_STANDARD})
```

Comment:
I would put this is the "less is more" bucket, i'd refrain from style changes like this 🙏